build.sh

```diff
@@ -35,6 +35,7 @@ usage()
   echo "  --source-version <SHA>          Source Link revision, required when building from tarball"
   echo "  --with-packages <DIR>           Use the specified directory of previously-built packages"
   echo "  --with-sdk <DIR>                Use the SDK in the specified directory for bootstrapping"
+  echo "  --auto-prep                     Automatically prepare the environment before building (replaces manual prep-source-build.sh call)"
```

Comment:
@tmds this is the new option.

Reply:
Consider dropping the `auto`, and just: `--prep`?

The explanation could emphasize on it downloading something: _Calls ./prep-source-build.sh to download bootstrap binaries_.

Reply:
Changed to --prep.

Speaking of odd ones, `-source-only|-source-build|-so|-sb` are too many aliases mapping to the same thing?